recipes/magics-metview/meta.yaml

```
@@ -0,0 +1,79 @@
+{% set name = "Magics" %}
+{% set version = "4.2.0" %}
+
+package:
+  name: magics-metview
+  version: {{ version }}
+
+source:
+  url: https://confluence.ecmwf.int/download/attachments/3473464/{{ name }}-{{ version }}-Source.tar.gz
+  sha256: d45701515d3329e1b3944d535c4d19f834d22a747a30da1d8c88b80affeb70cd
+  patches:
+    - 0001-Find-proj_6_0.lib-if-proj.lib-isn-t-found.patch
+    - 0002-Fix-for-Windows-build.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    # the CDTs below are needed when building with Metview enabled
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxfixes') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxcb') }}  # [linux]
+  host:
+    - pkg-config
+    - boost
+    - expat
+    - glib
+    - pango
+    - cairo
+    - proj
+    - eccodes
+    - libnetcdf
+    - jinja2
+    - qt
+    - zlib
+  run:
+    - python
+    - expat
+    - pango
+    - cairo
+    - proj
+    - eccodes
+    - simplejson
+    - libnetcdf
+    - qt
+    - zlib
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+  run_constrained:
+    # magics-metview is incompatible with magics
+    - magics ==9999999999
+
+test:
+  commands:
+    - printf "%s\n" "${MAGPLUS_HOME:?MAGPLUS_HOME is not set}"
+
+about:
+  home: https://software.ecmwf.int/wiki/display/MAGP/
+  license: Apache 2.0
```

The SPDX identifier has a dash:
```suggestion
  license: Apache-2.0
```
(just for consistency with other recipes)